### ISSUE TITLE
MODX browser breaks with custom filemanager_path

### ISSUE THREAD

**Issue by jgulledge19** [modxbot/migrate#4804]
jgulledge19 created Redmine issue ID 4804

I changed the main system setting, filemanager_path to "assets/content" and I also changed this for specific users.  This then breaks both image TVs and Tinymce from inserting the proper path.  It inserts something like "myfolder/pic.jpg" instead of "assets/content/myfolder/pic.jpg".  Basically the data.fullRelativeUrl and data.relativeUrl both are missing the user setting for filemanager_path.

_Here is a fix for this issue:_
In file  /core/model/modx/processors/browsers/directory/getfiles.php - about line 170:
Old: 'fullRelativeUrl' => rtrim($baseUrl).ltrim($dir.$fileName,'/'),
New: 'fullRelativeUrl' => rtrim($baseUrl).$modx->getOption('filemanager_path').ltrim($dir.$fileName,'/'),

This is also similar to issue here: http://bugs.modx.com/issues/4340 and should fix it.

**Comment by MarkH** [comment]
markh submitted:

I think you should set the base_path or $base_url setting, or at least that's what worked for me in rc3.

**Comment by jgulledge19** [comment]
jgulledge19 submitted:

Thanks Mark but there is no setting base_path in the System Settings, I looked in my install and a complete new install.  And I don't think this would solve the issue for users that have a different setting for filemanager_path unless you had to declare the base_path for every user.  To me that seems redundant and to many steps.

I added the code above and it now works fine for all of my users.  In my opinion this is a bug because at the very least it is not working as expected/desired and should not require creating some additional(and unknown) setting.

**Comment by MarkH** [comment]
markh submitted:

That a setting isn't included, doesn't mean you can't make it :P

I think I randomly came across it in some coding, and I've had it as a context setting. 

I've had this working without it as well some time ago, but don't know the exact settings I used. rb_base_dir comes to mind now.

**Comment by splittingred** [comment]
splittingred submitted:

This was fixed another way; your fix does not respect the working context. I can't replicate your situation in latest Git, so I'm assuming that it has been fixed in the myriad of commits since 2.1.0-rc4.

**Comment by lithiumlab** [comment]
lithiumlab submitted:

This is happening to me right now in 2.0.8-pl using IIS, windows 2003 server.
I've changed filemanager_path and TV images for example doesn't load the image in the manager.
All is going normal in the front end. they work, but are not visible in the manager until i choose another picture from the file browser for example. Tried also with different user system settings and it doesn't override.
